modules/account-lib/src/coin/baseCoin/enum.ts

```diff
@@ -29,6 +29,10 @@ export enum TransactionType {
   StakingWithdraw,
   // Handle smart contract calls
   ContractCall,
+  // Execute Dot Proxy,
+  Proxy,
+  // Create Dot Proxy,
+  AddProxy,
```

Comment:
If this is a proxy account initialization, then we should reuse `AddressInitialization`